docs/ref/checks.txt

```diff
@@ -446,7 +445,7 @@ The following checks are run if you use the :option:`check --deploy` option:
   set to ``True``, so your pages will not be served with an
   ``'X-Content-Type-Options: nosniff'`` header. You should consider enabling
   this header to prevent the browser from identifying content types incorrectly.
-* **security.W007**: Your :setting:`SECURE_BROWSER_XSS_FILTER` setting is not
+* **security.W007**: Your ``SECURE_BROWSER_XSS_FILTER`` setting is not
```

Comment:
Perhaps it is worth saying that the feature was removed in Django 4.0 to follow on from the check being removed in Django 3.0?

Reply:
I think it's fine to keep the current wording.